src/test/gitSecrets.test.ts

```diff
@@ -0,0 +1,108 @@
+/*!
+ * Copyright 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { describe } from 'mocha'
+
+import { spawnSync } from 'child_process'
+import * as assert from 'assert'
+import { getProjectDir } from './testUtil'
+import * as path from 'path'
+import { platform } from 'os'
+import { existsSync, mkdirSync, unlinkSync, writeFileSync } from 'fs'
+import dedent from 'ts-dedent'
+
+/**
+ * NOTES:
+ * - git-secrets patterns are set in the project's `.git/config` file
+ */
+describe('Ensure no secrets detected in project', function () {
```

Comment:
this gets prepended to the `it('ensures no git secrets are found'` below. so it's usually fine to keep the describe() titles very terse.

```suggestion
describe('git-secrets', function () {
```